pydarn/plotting/fan.py

```diff
@@ -268,6 +270,9 @@ def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
             rs - radius polar coordinates
         """
         # Get radar beam/gate locations
+        if ranges == []:
+            max_ranges = SuperDARNRadars.radars[stid].hardware_info.gates
+            ranges = [0, max_ranges]
```

Comment:
This part (L273-275) is unnecessary. By involving the hardware files, it allows the possibility for conflicts with the FOV files. Possible change suggested below.

Reply:
Well, that is rude? It is when you limit the FOV to being smaller than it really it is.